docusaurus.config.js

```diff
@@ -35,6 +35,14 @@ const config = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           editUrl: 'https://github.com/redstone-finance/redstone-docs/tree/main',
+          lastVersion: 'current',
```

Comment:
Do we really need this if we disable the versioning?

Reply:
Taken from documentation https://docusaurus.io/docs/versioning#configuring-versioning-behavior 😄 I think we don't disable versioning, it is more like overriding all versions to the current one